src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/PaidInFullStaffNotificationServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.cmc.claimstore.MockSpringTest;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailProperties;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.email.EmailData;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyMap;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class PaidInFullStaffNotificationServiceTest extends MockSpringTest {
+
+    private static final byte[] PDF_CONTENT = {1, 2, 3, 4};
+
+    @Captor
+    private ArgumentCaptor<String> senderArgument;
```

Comment:
This is not needed. Just compare the value directly in the `verify()` calls (with `eq()` for example)

Reply:
Hey @andrewwa-kainos could you expand on this please? Thanks

Reply:
You only need to capture an argument if you're then going to examine it. In this case you're capturing it on line 51 (at time of writing) but then not doing anything with it. You could remove this field and then either test its value (which would be new) with `eq(whateverItsMeantToBe)` or just leave it as `anyString()`

Reply:
Ah right gotcha cheers

Reply:
Done :) @andrewwa-kainos